README.md

```diff
@@ -1,5 +1,5 @@
 # Demos for the neuro-san library
-This repository contains a collection of demos for the neuro-san library.
+This repository contains a collection of demos for the [neuro-san library](https://github.com/leaf-ai/neuro-san).
```

Comment:
Also added a link to neuro-san library. We need to add some kind of intro here at some point.